README.md

```diff
@@ -105,33 +317,33 @@ username
 password
 ```
 
-# Step 3 :Prepare circuit to publish to Proof Market
+## Step 2: Prepare circuit to publish to Proof Market
 ```
 python3 scripts/prepare_statement.py -c=/root/tmp/zkllvm/build/examples/arithmetics_example.ll -o=arithmetic.json -n=arithmetic -t=placeholder-zkllvm
 ```
-
-# Step 4: See All Published circuits
+  
+## Step 4: See All Published circuits
 ```
 python3 scripts/statement_tools.py get
 ```
 
-# Step 5: Push a Bid
+## Step 5: Push a Bid
 ```
 python3 scripts/bid_tools.py push --cost <cost of the bid> --file <json file with public_input> --key <key of the statement> 
 ```
 
-# Step 6: Push an Ask
+## Step 6: Push an Ask
 ```
 python3 scripts/ask_tools.py push --cost <cost of the ask> --key <key of the statement> 
```

Comment:
```suggestion
## Step 5: Push a proposal
```
python3 scripts/proposal_tools.py push --cost <cost of the proposal> --key <key of the statement> 
```